scripts/change_file_ext_test.py

```diff
@@ -0,0 +1,164 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/change_file_ext.py."""
+
+import os
+import shutil
+
+import change_file_ext # pylint: disable=relative-import
```

Comment:
Please avoid pragma. There is already an issue for removing this pragma #5714.
I think absolute import will be fine.
Thanks

Reply:
@import-keshav don't you think that this is necessary in this case? Because the file is not a module, its a script and I think we can't have relative import for that. And those are the things you probably said in #5714. What do you think?

Reply:
Thanks, @YashJipkate, Yes you are right we can't use relative import in scripts that's why I told to do an absolute import.
I know ``` import change_file_ext ``` is an absolute import, but for this pylint throws an error but if we use full path absolute import like this ``` from scripts import change_file_ext ``` I think it will work fine.

Reply:
@import-keshav Are you sure about that? Because the command you mentioned is still a relative import. And it probably gives `ValueError: Attempted relative import in non-package`. Have you tried that?

Reply:
@YashJipkate, I tried it, it all works fine.
Can you tell me how its a relative import?
And @ankita240796 please ignore this review for a while.
Thanks

Reply:
Ah okay @import-keshav If it works fine then I have no worries! Thanks for checking :)
And if this works fine then @ankita240796 you should get rid of it.

Reply:
Done.